content/app/accessing-cloud-infra.md

```diff
@@ -54,9 +54,13 @@ To be able to impersonate the above service account, annotate your service accou
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: NAME
+  name: NAME #for the above examples this should be "sa" 
   annotations:
-    iam.gke.io/gcp-service-account: CLOUD_SERVICE_ACCOUNT
+    iam.gke.io/gcp-service-account: CLOUD_SERVICE_ACCOUNT #for the above example this will be of the form myfirsttenancy-ca@{{ project-id }}.iam.gserviceaccount.com
```

Comment:
again the `CLOUD_SERVICE_ACCOUNT` isn't helpful here, just replace it with what's in the comment